mail/templates/moderator_posts/subject.txt

```diff
@@ -0,0 +1 @@
+[{{ post.channel_title|safe }}] {{ post.title|safe }}
```

Comment:
Can you add "New Post on Open" or something like that as a prefix so it's easier for moderators to setup an email rule to sort these into a folder?

Reply:
@abeglova still seeing this needing to be fixed